--- bolt/exec/insights/tests/rules/ScanIoBoundRuleTest.cpp
+++ bolt/exec/insights/tests/rules/ScanIoBoundRuleTest.cpp
@@ -16,13 +16,13 @@
   PipelineStats pipeStats(true, false, false);
   bytedance::bolt::exec::OperatorStats opStats(1, 1, "node_1", "TableScan", false, {});
   
-  bytedance::bolt::exec::RuntimeMetric ioWait;
+  bytedance::bolt::RuntimeMetric ioWait;
   ioWait.sum = 100000000; // 0.1s
-  bytedance::bolt::exec::RuntimeMetric totalTime;
+  bytedance::bolt::RuntimeMetric totalTime;
   totalTime.sum = 2000000000; // 2s
   
   opStats.runtimeStats["ioWaitWallNanos"] = ioWait;
   opStats.runtimeStats["totalScanTime"] = totalTime;
   
   pipeStats.operatorStats.push_back(opStats);
   sample.taskStats.pipelineStats.push_back(pipeStats);
@@ -42,13 +42,13 @@
   PipelineStats pipeStats(true, false, false);
   bytedance::bolt::exec::OperatorStats opStats(1, 1, "node_1", "TableScan", false, {});
   
-  bytedance::bolt::exec::RuntimeMetric ioWait;
+  bytedance::bolt::RuntimeMetric ioWait;
   ioWait.sum = 1800000000; // 1.8s
-  bytedance::bolt::exec::RuntimeMetric totalTime;
+  bytedance::bolt::RuntimeMetric totalTime;
   totalTime.sum = 2000000000; // 2s
   
   opStats.runtimeStats["ioWaitWallNanos"] = ioWait;
   opStats.runtimeStats["totalScanTime"] = totalTime;
   
   pipeStats.operatorStats.push_back(opStats);
   sample.taskStats.pipelineStats.push_back(pipeStats);
